.eslintrc

```diff
@@ -2,11 +2,23 @@
   "parser": "babel-eslint",
   "plugins": [
     "async-await",
-    "flowtype"
+    "flowtype",
+    "import"
   ],
   "settings": {
     "flowtype": {
       "onlyFilesWithFlowAnnotation": true
+    },
+    "import/ignore": [
+      // Because of CommonJS incompatibility, we can't
+      // check for bad imports in node_modules.
+      "node_modules", 
```

Comment:
Huh, I forget what the exact issue was here in `addons-frontend`. I just removed it and `web-ext` has no complaints :) Let's remove this entire `import/ignore` block from the config for now until we run into specific problems.